src/cgnslib.h

```diff
@@ -1274,6 +1391,9 @@ CGNSDLL int cg_section_initialize(int file_number, int B, int Z, int S);
 CGNSDLL int cg_parent_data_write(int file_number, int B, int Z, int S,
 	const cgsize_t * parent_data);
 CGNSDLL int cg_npe( CGNS_ENUMT(ElementType_t) type, int *npe);
+CGNSDLL int cg_npe_ho( CGNS_ENUMT(ElementType_t) basicType, int order, int *npe);
```

Comment:
Settle on `int` for npe, even though a size is being returned. Matches cg_npe, and it should never exceed an int. Same with the other APIs that return a size.